docs/src/devdocs/elements.md

```diff
@@ -2,17 +2,20 @@
 
 ## Type definitions
 
-Elements or cells are subtypes of `AbstractCell{dim,N,M}`. They are parametrized by
-the dimension of their nodes via `dim`, the number of nodes `N` and the number
-of faces `M`.
+Elements or cells are subtypes of `AbstractCell{<:AbstractRefShape}`. As shown, they are parametrized
+by the associated reference element.
 
 ### Required methods to implement for all subtypes of `AbstractCell` to define a new element
 
 ```@docs
+Ferrite.reference_vertices(::Ferrite.AbstractRefShape)
 Ferrite.vertices(::Ferrite.AbstractCell)
+Ferrite.reference_edges(::Ferrite.AbstractRefShape)
 Ferrite.edges(::Ferrite.AbstractCell)
 Ferrite.reference_faces(::Ferrite.AbstractRefShape)
 Ferrite.faces(::Ferrite.AbstractCell)
+Ferrite.reference_facets(::Ferrite.AbstractRefShape)
+Ferrite.facets(::Ferrite.AbstractCell)
 Ferrite.geometric_interpolation(::Ferrite.AbstractCell)
 ```
```

Comment:
Actually, this isn't true AFAIU.
It suffices to implement `get_node_ids(c::MyCellType)` and `geometric_interpolation`. The part regarding reference shapes should be moved to reference shape devdocs, and stated here that this refers to a new cell for existing reference shape. 

```suggestion
**Move to refshapes devdocs**
### Required methods to implement for a new `AbstractRefShape`
```@docs
Ferrite.reference_vertices
Ferrite.reference_edges(::Ferrite.AbstractRefShape)
Ferrite.reference_faces(::Ferrite.AbstractRefShape)
```
which automatically defines 
```@docs
Ferrite.reference_facets(::Ferrite.AbstractRefShape)
```


### Required methods to implement for all subtypes of `AbstractCell` to define a new element

```@docs
Ferrite.get_node_ids
geometric_interpolation
```

### Automatically defined methods using the above required methods
```@docs
Ferrite.vertices(::Ferrite.AbstractCell)
Ferrite.edges(::Ferrite.AbstractCell)
Ferrite.faces(::Ferrite.AbstractCell)
Ferrite.facets(::Ferrite.AbstractCell)
```

Reply:
Oh right, I totally misread the headline.